frappe/public/js/frappe/list/list_sidebar_group_by.js

```diff
@@ -214,14 +214,13 @@ frappe.views.ListGroupBy = class ListGroupBy {
 	}
 
 	get_dropdown_html(field, fieldtype, applied = false) {
-		let label = field.name == null ? __("Not Set") : field.name;
+		let label = field.name == null ? __("Not Set") : __(field.name);
```

Comment:
I intentionally didn't suggest this in my last comment because the following conditions assume that label contains the untranslated value. 😉

Reply:
Yep. I thought about it, and check with different fields types ... seems ok but I agree. Changed.